### DISCUSSION TITLE
public org npm tokens

### DISCUSSION THREAD

**Discussion by rmarkins-godaddy** [npm/feedback#372] [General]
Currently in OS packages for NPM publishes via CICD a token is required for the most part. I'm wondering if it is possible to add public org NPM tokens to authenticate as the org and be able to publish via CICD pipeline and what that exp would look like in the NPM dashboard?

**Comment by MylesBorins** [comment]
Hey @rmarkins-godaddy. We currently have an [item on the roadmap](https://github.com/npm/roadmap/issues/10) to explore package-scoped tokens. This is something that we would need to scope and implement before we would start exploring org based tokens (for a number of reasons including overall complexity).

For now I don't think org-based tokens is something that we would prioritize on our roadmap, but this is something that we could revisit in the future based on our solution for package-scoped tokens.

**Reply by rmarkins-godaddy** [reply]
thanks for the response @MylesBorins. This makes sense and solves our opensource packages solution. Thanks for pointing out the issue. I'll keep a watchful eye on it 😄
